jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStep.java

```diff
@@ -78,7 +76,7 @@ public class WithMavenStep extends Step {
     private String mavenLocalRepo = "";
     private List<MavenPublisher> options = new ArrayList<>();
     private MavenPublisherStrategy publisherStrategy = MavenPublisherStrategy.IMPLICIT;
-    private boolean traceability = true;
+    private boolean traceability = false;

```

Comment:
Did you accidentally mix b688eb87db0a9e85b6c0364e874912f8310de8ee into an unrelated PR?

Reply:
I needed a fix done on my other open PR for test to pass.

I have extracted this fix to a dedicated commit on master, and rebased my two PR on top of it.

Should be fine now